nncf/quantization/precision_init/hawq_debug.py

```diff
@@ -105,30 +185,9 @@ def get_bitwidth_graph(algo_ctrl, model, all_quantizers_per_full_scope,
 
         non_weight_quantizers = algo_ctrl.non_weight_quantizers
         bits_color_map = {4: 'red', 8: 'green', 6: 'orange'}
-        for quantizer_id, quantizer_info in non_weight_quantizers.items():
-            affected_iap_ctx_list = quantizer_info.affected_ia_op_exec_contexts
-            for activation_iap_ctx in affected_iap_ctx_list:
-                post_hooked_nx_node_key = nncf_graph.get_node_id_by_iap_context(activation_iap_ctx)
-                post_hooked_module_node = nncf_graph.get_nx_node_by_key(post_hooked_nx_node_key)
-                operator_name = post_hooked_module_node[NNCFGraph.OP_EXEC_CONTEXT_NODE_ATTR].operator_name
-                if not grouped_mode:
-                    operator_name += '_#{}'.format(str(post_hooked_module_node[NNCFGraph.ID_NODE_ATTR]))
-                post_hooked_module_node['label'] = operator_name
-
-                for next_nx_node_key in nncf_graph.get_successors(post_hooked_nx_node_key):
-                    activation_fq_node = nncf_graph.get_nx_node_by_key(next_nx_node_key)
-                    activation_quantizer = non_weight_quantizers[quantizer_id].quantizer_module_ref
-                    bits = activation_quantizer.num_bits
-
-                    activation_fq_node['color'] = bits_color_map[bits]
-                    activation_fq_node['style'] = 'filled'
-                    node_id = activation_fq_node[NNCFGraph.ID_NODE_ATTR]
-                    if grouped_mode:
-                        node_id = groups_of_adjacent_quantizers.get_group_id_for_quantizer(activation_quantizer)
```

Comment:
please add a group number to the node name. It was very useful for debugging purposes

Reply:
But it's still there, new line 155, is it not?

Reply:
yes, this code is ok. For some reason, `groups_of_adjacent_quantizers` is empty now, which is not expected. Probably, this is also the reason, why graphs look incorrect